tractdb_pyramid/views/storytelling_stravaview.py

```diff
@@ -16,6 +16,14 @@
     cors_credentials=True
 )
 
+service_access_token = cornice.Service(
+    name='storytelling_strava_access_token',
+    path='/storytelling/strava/access_token/{code}',
```

Comment:
This is weird and not RESTful.

The `code` is content, not an identifier, so it shouldn't be in the URI. Should be in the body of the request instead. For example, see how the `authenticationview.py` accesses `json_body` in `login_post`.

This should also be a `put`, not a `post`. You're creating or modifying a specific identifier.